test/sslapitest.c

```diff
@@ -9015,7 +9015,7 @@ static int test_session_cache_overflow(int idx)
          * Cause this session to have a longer timeout than the next session to
          * be added.
          */
-        if (!TEST_true(SSL_SESSION_set_timeout(sess, LONG_MAX))) {
+        if (!TEST_true(SSL_SESSION_set_timeout(sess, LONG_MAX / 2))) {
```

Comment:
Would it be useful to define `OSSL_TIME_MAX` along the lines of `OSSL_TIME_SECOND` and friends in `internal/time.h`?

Reply:
Ah no, `SSL_SESSION_set_timeout()` takes a number of seconds, not an `OSSL_TIME`

Reply:
That wouldn't actually solve the problem here. The max value is fine (this function does actually take a `long`) but it causes a problem when we add the current time to it (as described in #24046). I'd prefer not to overload this PR any more with possible solutions. I'd suggest we consider any subsequent changes as part of any response to #24046.